components/SearchResult.tsx

```diff
@@ -1,24 +1,29 @@
-import { useQuery } from "@tanstack/react-query";
+import { useInfiniteQuery } from "@tanstack/react-query";
 import Result from "./Result";
 import { Album, SearchResponseSchema } from "@/schemas/collections.schemas";
 import ResultLoading from "./ResultLoading";
 
 export default function SearchResult({ searchValue }: { searchValue: string }) {
-  const { data, isError, isLoading } = useQuery({
+  const { data, error, isFetching, fetchNextPage } = useInfiniteQuery({
     queryKey: ["search", searchValue],
-    queryFn: async () => {
+    queryFn: async ({ pageParam }) => {
       const params = new URLSearchParams({
-        search: searchValue,
+        search: searchValue ? searchValue : "flume",
```

Comment:
```suggestion
        search: searchValue,
```

Reply:
Removed but would like to handle my front page like this. Not "flume" necessarily but maybe something else